ts/features/bonus/cgn/components/merchants/CgnDiscountDetail.tsx

```diff
@@ -142,6 +149,20 @@ const CgnDiscountDetail: React.FunctionComponent<Props> = ({
           <H4 weight={"Regular"}>{discount.condition}</H4>
         </>
       )}
+      {discount.landingPageUrl && discount.landingPageReferrer && (
+        <ButtonDefaultOpacity
+          style={{ width: "100%" }}
+          onPress={() => {
+            onLandingCtaPress?.(
+              discount.landingPageUrl as string,
+              discount.landingPageReferrer as string
+            );
+          }}
+          onPressWithGestureHandler={true}
+        >
+          <Label color={"white"}>{"Accedi all’agevolazione"}</Label>
```

Comment:
"Accedi all’agevolazione" should be translated

Reply:
Yep missed it ty!